src/pages/Events.jsx

```diff
@@ -4,6 +4,7 @@ import Grid from '@material-ui/core/Grid';
 import { withStyles } from '@material-ui/core/styles';
 import { Paper, Typography } from '@material-ui/core';
 import { connect } from 'react-redux';
+import CircularProgress from '@material-ui/core/CircularProgress';
```

Comment:
Good Idea to add Loading Icons.
Since we are using CircularProgress at many places, it's better to add it as a Component and use that Component.

Reply:
Resolved in 97de9971939c449077ef5999058a7b244e275661